mgmt_cli_test.py

```diff
@@ -710,9 +751,10 @@ def _insert_data_while_excluding_each_node(self, total_num_of_rows, keyspace_nam
         for node in self.db_cluster.nodes:
             node.run_nodetool("flush")
 
-    def create_missing_rows_in_cluster(self, create_missing_rows_in_multiple_nodes, keyspace_to_be_repaired=None):
+    def create_missing_rows_in_cluster(self, create_missing_rows_in_multiple_nodes, keyspace_to_be_repaired=None,
+                                       total_num_of_rows=29296872):
```

Comment:
please don't put a default like that, 
put it in the test only, and put a comment where it's taken from how it was calculated, and such.